misc/zsh_completion/git-bug

```diff
@@ -17,6 +17,12 @@ case $state in
   ;;
   level2)
     case $words[2] in
+      status)
```

Comment:
Please don't commit that file. It's a generated file but the generator is not determinist (should be fixed now).

Reply:
OK